packages/statemanager/src/statelessVerkleStateManager.ts

```diff
@@ -903,21 +896,26 @@ export class StatelessVerkleStateManager implements EVMStateManagerInterface {
   async flush(): Promise<void> {}
 
   /**
-   * Gets the verkle root.
-   * NOTE: this needs some examination in the code where this is needed
-   * and if we have the verkle root present
-   * @returns {Promise<Uint8Array>} - Returns the verkle root of the `StateManager`
+   * Gets the cache state root.
+   * This is used to persist the stateRoot between blocks, so that blocks can retrieve the stateRoot of the parent block.
```

Comment:
This will only work for block execution when you're doing it sequentially, right?

Reply:
Indeed. We could come up with a more elaborate scheme, however Guillaume confirmed to me that the verkle block format would be updated to also contain the preStateRoot. We therefore will not need this in the next testnet, so I think that this naive approach is sufficient for testing purposes.